pyfixest/did/did2s.py

```diff
@@ -82,13 +87,17 @@ def __init__(
         self._first_u = np.array([])
         self._second_u = np.array([])
 
+        # column name with weights. None by default
+        self._weights = weights
```

Comment:
For consistency with the `Feols` class, could we call this `self._weights_name`?

Reply:
Actually, not really a must have 🤔

Reply:
Hey, definitely good to add consistency. I'm going to add it.